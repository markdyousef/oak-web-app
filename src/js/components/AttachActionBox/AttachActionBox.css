.container {
    position: absolute;
    width: 250px;
    background-color: #fff;
    z-index: 2000;
    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
    display: flex;
    flex-direction: column;
    align-items: center;
}

.container header {
    width: 100%;
    border-bottom: 1px solid #ddd;
    height: 30px;
    margin: 10px;
    display: flex;
    align-items: center;
    justify-content: center;
    padding: 10px;
}

.container header h3 {
    font-size: 12px;
    color: rgb(156, 156, 156);
}

.link {
    margin-top: 20px;
    width: 80%;
}
