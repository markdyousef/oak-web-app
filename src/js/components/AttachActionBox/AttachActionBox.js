// @flow
import React, { Component, PropTypes } from 'react';

import css from './AttachActionBox.css';

class AttachActionBox extends Component {
    static propTypes = {}
    static defaultProps = {}
    constructor() {
        super();
        this.state = {};
    }
    render() {
        return (
            <div className={css.container}>

            </div>
        );
    }
}

export default AttachActionBox;
