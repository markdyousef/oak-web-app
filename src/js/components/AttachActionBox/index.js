import AttachActionBox from './AttachActionBox';

export default AttachActionBox;
