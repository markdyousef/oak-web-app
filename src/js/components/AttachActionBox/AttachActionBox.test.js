import React from 'react';
import renderer from 'react-test-renderer';
import AttachActionBox from './AttachActionBox';

it('renders correctly', () => {
    const props = {
        close: () => {}
    };
    const component = renderer.create(
        <AttachActionBox {...props} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
