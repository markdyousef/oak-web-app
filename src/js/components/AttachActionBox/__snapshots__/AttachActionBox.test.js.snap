exports[`test renders correctly 1`] = `
<div
  className={undefined}>
  <header>
    <h3>
      Attach from...
    </h3>
  </header>
  <div
    className={undefined}>
    <div
      className={undefined}>
      <input
        onChange={[Function]}
        placeholder="www.clai.io"
        type="text"
        value="" />
      <span
        className={undefined} />
      <span
        className={undefined} />
      <label
        htmlFor="input">
        URL
      </label>
    </div>
  </div>
  <div>
    <button
      className=" "
      onClick={[Function]}>
      <div>
        <label
          className={undefined}
          htmlFor="text">
          Attach
        </label>
      </div>
    </button>
  </div>
</div>
`;
